src/modules/simulation/simulator_mavlink/CMakeLists.txt

```diff
@@ -71,3 +71,28 @@ add_custom_target(none_iris
 		${PX4_SOURCE_DIR}/ROMFS/px4fmu_common/init.d-posix/airframes/10016_none_iris
 	COMMENT "launching px4 none_iris (SYS_AUTOSTART=10016)"
 )
+
+
+# xplane_c172 (legacy compatibility launch helper)
+add_custom_target(xplane_c172
```

Comment:
maybe move these to a dedicated file like the others (eg sitl_targets_jsbsim.cmake, etc).

Reply:
Hello again
I just wanted to have an update on this pull request. should I do anything else before it merges? Since I want to create youtube tutorials on this topic I prefer not to instruct people to use my forked repo. I am waiting until this PR merge then create tutorial based on this.
thanks in advance